### PR TITLE
Android tablet page offset artifact fix

### PR DESCRIPTION
**Pull request by sasf54**
Hi,

I had some trouble doing some sales presentations on my android 5.01 tablet with the color picker.
It had some issues with zooming: it was not able to place to the right place the colorpicker box, and was selecting off colors.
I made a fix, sorry for the styling mishap :(
(detects if it's an android, and only then applies fixes)
relevant codes:
default variables:
android:

functions / event handlers:
reposition:
mousedown:

I hope it will be useful for others too.

Bests,
Ferenc

### CODE REVIEW

**Comment by itsjavi** [comment]
Thanks, but one does not simply change the code style of a whole project, this is very hard to follow and review.
Please, could you do only the necessary changes, generate the minified files and rebase your commit?
Thank you

**Comment by sasf54** [comment]
Hi Javi,

Committed with corrected formatting.
(Sorry, I'm a bit new to github contributing, and I can't import the
formatting specification into my IDE)

Bests,
Ferenc

**Comment by itsjavi** [comment]
No problem @sasf54 we all have been new some day ;)

But can you squash your commits into a single one? So I can properly review this
thanks

**Comment by sasf54** [comment]
Hi Javi,

Sorry, that's the best I can do :(
(it's in two commits, and the final state are definitely comparable with
the master branch...)

Bests,
Ferenc
